modules/kubernetes-addons/cluster-autoscaler/locals.tf

```diff
@@ -41,7 +41,6 @@ locals {
     create_kubernetes_service_account = true
     kubernetes_service_account        = local.service_account_name
     irsa_iam_policies                 = [aws_iam_policy.cluster_autoscaler.arn]
-    irsa_iam_permissions_boundary     = var.irsa_iam_permissions_boundary
   }
```

Comment:
Do you want to use the addon_context var here or not relevant?

Reply:
No this is now passed transparently to irsa module via addon_context